README.md

```diff
@@ -79,16 +93,24 @@ Deploy a smart contract from an account.
 
 Deploy a smart contract to the network.
 ```
+If you like to check the balance on Bob's account use the following command:
 
-## I still need help
-
-[Please reach out to me](mailto:jeff@immutability.io). 
+```sh
+    vault read vault-eth2/accounts/bob/balance
+```
 
-## Tip
+Running this command we expect an output that provides the account address for Bob as well as its balance:
 
-Supporting OSS is very hard. 
+```sh
+    Key        Value
+    ---        -----
+    address    0x90259301a101A380F7138B50b6828cfFfd4Cbf60
+    balance    999496453868000000000
+```
 
-This is my ETH address. The private keys are managed by this plugin: 
+## If you still need help or have a question about how things work
+> [!NOTE]
+> This repository was updated by Asma Taamallah, working student at the FIWARE Foundation and student at the University of Passau, Germany.
+> If you still need help or have a question about how things work, [Please fell free to reach out](mailto:asma.taamallah@fiware.org). 
```

Comment:
Due to we fork the original repo, I suggest to keep the reference to the original person.